### PR TITLE
Updates to campaign loadout UI

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Updates the UI a bit.

Slightly different layout for loadout options/now with a scroll bar.

Awesome new icons from GregorMN. They are not all complete so there are still older/placerholder ones.

image
## Why It's Good For The Game
Easier to navigate/read UI.
## Changelog
:cl:
qol: Campaign: Improved loadout UI and new icons
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx [hunk 1]
```diff
@@ -128,69 +128,71 @@ export const IndividualLoadouts = (props) => {
...
+          <Section fill scrollable width={'240px'} height={'555px'}>
+            <Flex width={'220px'} wrap>
+              {available_loadouts_data
+                .filter((potentialitem) => potentialitem.job === selectedJob)
+                .filter(
+                  (potentialitem) =>
+                    potentialitem.slot === selectedLoadoutItem.item_type.slot,
+                )
+                .map((potentialitem) => (
+                  <Flex.Item mr={1.5} mb={1} key={potentialitem.name}>
```
Comment (TiviPlus) [inline tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx:151]: we use stack not flex, usually why are you using flex here, you're adding padding anyway

File: tgui/packages/tgui/interfaces/IndividualStats/index.tsx [hunk 2]
```diff
@@ -98,8 +98,8 @@ export const IndividualStats = (props) => {
     <Window
       theme={data.ui_theme}
       title={'Prep screen'}
-      width={850}
-      height={750}
+      width={980}
+      height={775}
```
Comment (TiviPlus) [inline tgui/packages/tgui/interfaces/IndividualStats/index.tsx:102]: probably not an issue yet but be careful with UI sizes not everyone has a big monitor

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/IndividualStats/index.tsx (hunk 2 above)
Comment (Lumipharon) [reply tgui/packages/tgui/interfaces/IndividualStats/index.tsx:102]: I don't think it should get any bigger than this now. Who the heck has anything less than 1280x720 tho?

File: tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx (hunk 1 above)
Comment (Lumipharon) [reply tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx:151]: As mentioned in discord, stack screws up the formatting. Happy to change it if you know how to make it work with stack though.

Comment (Lumipharon) [reply tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx]: done

File: tgui/packages/tgui/interfaces/IndividualStats/index.tsx (hunk 2 above)
Comment (TiviPlus) [reply tgui/packages/tgui/interfaces/IndividualStats/index.tsx:99]: have you SEEN how zack plays

**Pull request merged**